packages/concerto-core/test/dcsIndexWrapper.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const DcsIndexWrapper = require('../lib/DcsIndexWrapper');
```

Comment:
@mttrbrts Coverage for the file this test targets at is showing 0%. Is there anything more I should be doing if I add a new test suite?

Reply:
Do you see the same locally, as in the GH Actions build?

Reply:
Locally

Reply:
Moved to `DecoratorManager.js` fixed the issue